templates/packages.erb

```diff
@@ -1 +1,42 @@
-asciidoc bash bzip2 cmake3 cowsay emacs-common fail2ban figlet fortune-mod gcc httpd httpd-itk ipset iptables kernel mariadb mariadb-server memcached mg nano net-tools nginx ntp ntpdate openssh openssh-server openssl php php-mysql postgresql postgresql-server ruby rubygems screen socat sysstat tcpdump vim xmlto zlib zlib-devel zsh 
+asciidoc
```

Comment:
Does this work the same way? I thought I tried doing this and it didn't like it.